hack/quickstart/init-master.sh

```diff
@@ -38,15 +39,26 @@ function init_master_node() {
     systemctl stop update-engine; systemctl mask update-engine
 
     # Start etcd.
-    configure_etcd
-    systemctl enable etcd-member; sudo systemctl start etcd-member
+    if [ "$SELF_HOST_ETCD" = false ] ; then
+        configure_etcd
+        systemctl enable etcd-member; sudo systemctl start etcd-member
+    fi
 
     # Render cluster assets
-    /usr/bin/rkt run \
-        --volume home,kind=host,source=/home/core \
-        --mount volume=home,target=/core \
-        --trust-keys-from-https --net=host ${BOOTKUBE_REPO}:${BOOTKUBE_VERSION} --exec \
-        /bootkube -- render --asset-dir=/core/assets --api-servers=https://${COREOS_PUBLIC_IPV4}:443,https://${COREOS_PRIVATE_IPV4}:443
+    if [ "$SELF_HOST_ETCD" = true ] ; then
+        /usr/bin/rkt run \
+            --volume home,kind=host,source=/home/core \
+            --mount volume=home,target=/core \
+            --trust-keys-from-https --net=host ${BOOTKUBE_REPO}:${BOOTKUBE_VERSION} --exec \
+            /bootkube -- render --asset-dir=/core/assets --api-servers=https://${COREOS_PUBLIC_IPV4}:443,https://${COREOS_PRIVATE_IPV4}:443 \
+            --etcd-servers=http://10.3.0.15:2379  --experimental-self-hosted-etcd
+    else
+        /usr/bin/rkt run \
+            --volume home,kind=host,source=/home/core \
+            --mount volume=home,target=/core \
+            --trust-keys-from-https --net=host ${BOOTKUBE_REPO}:${BOOTKUBE_VERSION} --exec \
+            /bootkube -- render --asset-dir=/core/assets --api-servers=https://${COREOS_PUBLIC_IPV4}:443,https://${COREOS_PRIVATE_IPV4}:443
```

Comment:
Most of this is the same between both cases, the only differences is the flags. Could simplify just by parameterizing the flags themselves (e.g. https://github.com/kubernetes-incubator/bootkube/blob/master/hack/multi-node/bootkube-up#L12-L20)